scripts/aggregate-translations/aggregate-translations-cli.js

```diff
@@ -5,33 +5,22 @@ const parseCLIList = require('../utils/parse-cli-list');
 
 const aggregateTranslations = require('./aggregate-translations');
 
-// Adds custom search directory paths
-const customSearchDirectories = [];
-const addCustomDirectory = (searchPattern) => {
-  customSearchDirectories.push(searchPattern);
-};
-
-const customExcludeDirectories = [];
-const addCustomExclude = (searchPattern) => {
-  customExcludeDirectories.push(searchPattern);
-};
-
 // Parse process arguments
 commander
   .version(i18nPackageJson.version)
-  .option('-b, --baseDir [baseDir]', 'The directory to start searching from and to prepend to the output directory', process.cwd())
-  .option('-d, --directories [directories]', 'Regex pattern to glob search for translations', addCustomDirectory)
-  .option('-l, --locales [locales]', 'The list of locale codes aggregate on and combine into a single, respective translation file ', parseCLIList, supportedLocales)
-  .option('-o, --outputDir [outputDir]', 'The output location of the generated configuration file', './aggregated-translations')
-  .option('-c, --config [configPath]', 'The path to the terra i18n configuration file', undefined)
-  .option('-e, --exclude [exclude]', 'Regex pattern to glob filter out directories', addCustomExclude)
-  .option('-f, --format [format]', 'Format to output the aggregated translations to. Options are [es5, es6]', 'es5')
+  .option('-b, --baseDir [baseDir]', `The directory to start searching from and to prepend to the output directory. Default: ${process.cwd()}`)
+  .option('-d, --directories [directories]', 'Regex pattern to glob search for translations. Default: []', parseCLIList)
```

Comment:
Previously directories could be added like:
```bash
tt-aggregate-translations -d './translations' -d './custom_translations'
```
Does passing to parse parseCLIList change this behavior to:
```bash
tt-aggregate-translations -d ['./translations','./custom_translations']
```

Reply:
If this changes the CLI expectation, we might need to leave it as is until a MVB

Reply:
💯I agree. I shall revert the changes around these.